django/core/cache/backends/dummy.py

```diff
@@ -25,19 +25,10 @@ def delete(self, key, version=None):
         key = self.make_key(key, version=version)
         self.validate_key(key)
 
-    def get_many(self, keys, version=None):
-        return {}
-
     def has_key(self, key, version=None):
         key = self.make_key(key, version=version)
         self.validate_key(key)
         return False
 
-    def set_many(self, data, timeout=DEFAULT_TIMEOUT, version=None):
-        pass
```

Comment:
I'd use `return []` for the initial commit and have a separate commit that does the method removals as that's really a separate issue that's less needed for a backport to 2.0.